Python/ogusa/TPI.py

```diff
@@ -642,6 +645,8 @@ def run_TPI(income_tax_params, tpi_params, iterative_params, initial_values, SS_
     print 'Max Euler error, savings: ', eul_savings
     print 'Max Euler error labor supply: ', eul_laborleisure
 
+    if (np.absolute(eul_savings) >= mindist_TPI) or (np.absolute(eul_savings) > mindist_TPI):
+        raise RuntimeError("Transition path equlibrium not found")
```

Comment:
Need a "np.absolute(eul_laborleisure) > mindist_TPI" as one of the arguments in the conditional.